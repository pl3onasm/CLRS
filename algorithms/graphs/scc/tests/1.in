a b
b e
e a
e f
f g
g f
g h
h h
b c
c d
d c
d h
c g
b f
