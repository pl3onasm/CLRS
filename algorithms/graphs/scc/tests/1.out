
--------------------
 Graph
 Directed
 Unweighted
 Vertices: 8
 Edges: 14
--------------------
  a[1]: b
  b[3]: e, c, f
  c[2]: d, g
  d[2]: c, h
  e[2]: a, f
  f[1]: g
  g[2]: f, h
  h[1]: h
--------------------


Strongly connected components
-----------------------------
1: {h}
2: {g, f}
3: {d, c}
4: {b, a, e}
-----------------------------

